resnet/yamls/mcloud_run.yaml

```
@@ -0,0 +1,95 @@
+run_name: resnet_example
+cluster: null  # Name of the cluster to use for this run
+gpu_type: null # Type of GPU to use
+gpu_num: null  # Number of GPUs to use specified as an integer
```

Can these be sane defaults? 
```suggestion
gpu_type: a100_40 # Type of GPU to use
gpu_num: 8  # Number of GPUs to use
```

---

I was thinking about the [mcli examples](https://mcli.docs.mosaicml.com/getting_started/first_composer.html) where they don't specify gpu_type, but it's probably fine